src/legacy/core_plugins/vis_type_timelion/public/components/panel.tsx

```diff
@@ -70,18 +70,18 @@ const emptyCaption = '<br>';
 function Panel({ interval, seriesList, renderComplete }: PanelProps) {
   const kibana = useKibana<TimelionVisDependencies>();
   const [chart, setChart] = useState(() => cloneDeep(seriesList.list));
-  const [canvasElem, setCanvasElem] = useState();
-  const [chartElem, setChartElem] = useState();
+  const [canvasElem, setCanvasElem] = useState<HTMLElement>();
```

Comment:
the file still contains some type errors. @timroes  could someone take a look and fix the errors, please?